Public/Src/Cache/ContentStore/Distributed/NuCache/ContentLocationDatabase.cs

```diff
@@ -308,8 +311,8 @@ protected IEnumerable<ShortHash> EnumerateSortedKeys(OperationContext context)
             if (IsInMemoryCacheEnabled)
             {
                 // If we don't check, then GC will happen anyways and we will -incorrectly- increment the counter
-                Counters[ContentLocationDatabaseCounters.NumberOfCacheFlushesTriggeredByGarbageCollection].Increment();
-                ForceCacheFlush(context);
+                ForceCacheFlush(context,
+                    counter: ContentLocationDatabaseCounters.NumberOfCacheFlushesTriggeredByGarbageCollection);

```

Comment:
Based on the client of this methods, it seems that the counter is not correct.

This method is not called from the GC and only called from the worker machine.

Could you please double check that?